composer.json

```
@@ -4,7 +4,7 @@
     "description": "Temporary storage adapter factory for fluent migration to laminas-cache v3 when working with laminas components which depend on laminas-cache",
     "license": "BSD-3-Clause",
     "require": {
-        "php": "~8.0.0 || ~8.1.0 || ~8.2.0 || ~8.3.0",
+        "php": "~8.0.0 || ~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0",
```

[Version 8.0 is no longer supported:](https://www.php.net/supported-versions)
```suggestion
        "php": "~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0",
```